bears/general/QuotesBear.py

```diff
@@ -30,13 +31,22 @@ def correct_single_line_str(self, filename, file, sourcerange,
             The sourcerange indicating where to find the string.
         :param preferred_quotation:
             ``'`` or ``"`` respectively.
+        :param force_preferred_quotation:
+            Decide whether the preffered quotes are compulsory
+            to adhere or not.
         """
         str_contents = file[sourcerange.start.line - 1][
                        sourcerange.start.column:sourcerange.end.column-1]
 
-        if preferred_quotation in str_contents:
+        if (preferred_quotation in str_contents and
+                not force_preferred_quotation):
             return
 
+        # Escape preferred quotes if present.
+
+        escape_str = '\\' + preferred_quotation
```

Comment:
I would still inline this one ;)

Reply:
What does inline mean?

Reply:
You mean `str_contents = str_contents.replace(preferred_quotation, '\\' + preferred_quotation)`